feat(learn): Data Analysis Example B - Using the 'Loc' Method (#38656)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/data-analysis-example-b.english.md

```yml
question:

>   text: 'What does the 'loc' method allow you to do?

Reviewer: It might be nice to format this as code. Plus this has missing single quotes that causes issues.

```suggestion
  text: 'What does the <code>loc</code> method allow you to do?'
```

  answers:
    - 'Retrieve a subset of rows and columns by supplying interger-location arguments.'
    - 'Access a group of rows and columns by supplying label(s) arguments.'

>     - 'Returns the first n rows based on the integer argument supplied.'

Reviewer: ```suggestion
    - 'Returns the first <code>n</code> rows based on the integer argument supplied.'
```

>   solution: 3

Reviewer: Given the answers that were created, this solution should be changed to 2, not 3.

Reviewer 2: Thanks - moved answers around/deleted some things. 

Thanks for notes on code formatting, will use it in the future!

>   solution: 3

Reviewer 3: ```suggestion
  solution: 2
```
